lms/djangoapps/courseware/views/index.py

```diff
@@ -382,6 +382,7 @@ def _create_courseware_context(self, request):
             'section_title': None,
             'sequence_title': None,
             'disable_accordion': COURSE_OUTLINE_PAGE_FLAG.is_enabled(self.course.id),
+            'chapter_position': self.course.position
```

Comment:
`self.course` is already available in the context, so maybe we should just use that existing context in the template, rather than adding a new bit of context?

Reply:
good point, added course.position and chapter position to the dom